### DISCUSSION TITLE
Mute applications in the system tray

### DISCUSSION THREAD

**Discussion by adrianlopezgalera** [linuxmint/discussions#71] [Ideas]
Hello. One very cool feature of KDE Plasma is that it allows you to mute specific apps by clicking on a speaker icon next to the app name while it is in the system tray.

In short, my proposal is about incorporating a kind of shortcut to the volume control by applications that already exists in the corresponding menu. This option, perhaps, can be configurable in the panel settings.

What do you think about this idea?

Regards.

**Comment by Nebula-0-System** [comment]
i'd love if the devs could add some way to make a shortcut to edit the current application's volume, like holding alt while turning an audio control dial or something on the keyboard, or just the ability to make a shortcut like that. like the ability to add per-application volume control shortcuts that target the open application, to avoid needing to open the applet that Cinnamon usually has, maybe default bind it to holding alt and using a volume dial.

maybe this should be directed instead to pulseaudio for them to (up? down?)-stream these features to the applet in Cinnamon, but i'd really like this to be a feature, so if this is the case, please lemme know so i can send the idea to them as well.
